website/docusaurus.config.js

```diff
@@ -116,18 +116,14 @@ const config = {
           {
             to: '/docs/podman/installing',
             from: [
-              '/docs/onboarding-for-containers/installing-podman',
-              '/docs/onboarding/containers/installing-podman',
+              '/docs/Installation/windows-install/installing-podman-with-openshift-local',
```

Comment:
Is the Installation uppercase expected ?

Reply:
Yes, because we had it in the past.